src/components/Card/Card.tsx

```diff
@@ -244,7 +244,7 @@ export const Card: React.FC<React.PropsWithChildren<any>> = chakra(
       backgroundColor && (customColors["backgroundColor"] = backgroundColor);
       foregroundColor && (customColors["color"] = foregroundColor);
 
-      const styles = useMultiStyleConfig("Card", {
+      const styles = useMultiStyleConfig("CustomCard", {
```

Comment:
Let's use `ReservoirCard`. I know some values still use "Custom" but those can be updated in a "clean up" pr soon.